Seminar-IS/Seminar02/Pract 02/task01.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <fstream>
+
+using namespace std;
+
+void saveNumbersToFile() {
+    int* nums = new int[3];
```

Comment:
Do not allocate dynamic memory for arrays with such small number of elements. It is very slow operation. Do not allocate memory when you know the exact number of elements.